models/tasks.js

```diff
@@ -701,6 +701,20 @@ const markUnDoneTasksOfArchivedUsersBacklog = async (users) => {
   }
 };
 
+const fetchIncompleteTaskForUser = async (user) => {
+  const COMPLETED_STATUSES = [DONE, COMPLETED];
+  try {
+    const incompleteTaskForUser = await tasksModel
```

Comment:
Can't we just pass id no need to pass user object right?

Reply:
Yes, I'll make the changes.